cms/static/sass/elements/_modal-window.scss

```diff
@@ -579,28 +579,33 @@
   .wrapper-modal-window-bulkpublish-subsection,
   .wrapper-modal-window-bulkpublish-unit,
   .course-outline-modal {
-    .exam-time-list-fields,
```

Comment:
Note to reviewers: exam-time-list-fields and exam-review-rules-list-fields aren't used anywhere in the codebase.